llpc/util/llpcElfWriter.cpp

```diff
@@ -735,6 +737,288 @@ template <class Elf> void ElfWriter<Elf>::updateMetaNote(Context *pContext, cons
   pNewNote->data = data;
 }
 
+// =====================================================================================================================
+// Get the number of relocs in the reloc section
+//
+// @param relocSection : Reloc section
+// @returns : Return the number of the relocs
+template <class Elf> size_t ElfWriter<Elf>::numRelocs(const SectionBuffer *relocSection) {
+  if (relocSection->secHead.sh_size == 0)
+    return 0;
+
+  return static_cast<unsigned>(relocSection->secHead.sh_size / relocSection->secHead.sh_entsize);
+}
+
+// =====================================================================================================================
+// Get the start pos of relocs used by PS in the reloc section
+//
+// @param relocSection : Reloc section
+// @param psIsaOffset : Ps ISA offset in the elf bin
+// @returns : Return the start pos of the relocs used by PS in the reloc section
```

Comment:
nit: You can skip 'Return', I'd just start it with `@returns: Start pos of the...`. Or if you prefer to keep it, change it to 'Return**s**'.

Reply:
Also elsewhere

Reply:
Sure.